travis-dist-files.txt

```diff
@@ -0,0 +1,15 @@
+--exclude dist/static/
+dist/
+etc/
+examples/
+--exclude lib/storage/data/
+lib/
+--exclude node_modules/.cache/
+--exclude node_modules/monaco-editor/
+node_modules/
+static/policies/
+static/favicon.ico
+views/
+app.js
+package-lock.json
```

Comment:
Do we need these in the dist? I guess it doesn't matter (and may be useful debugging down the line)

Reply:
Got it :)